mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = (req, res) => {
+  Order.remove({ _id: req.params.orderId }) 
+    .exec((error, result) => {
+      if (error) {
+        return res.status(200).json({ 
+          success: "Order deletion failed", 
```

Comment:
Inconsistent HTTP status code. In the `deleteOrder` function, a 200 status code is returned when the order deletion fails, which is misleading. A 400 or 500 status code would be more appropriate to indicate an error. Additionally, the error message should be more descriptive to help with debugging and user experience.